SEP-0002.md

```diff
@@ -138,15 +137,27 @@ ED shall serve a one year term, and can be re-elected.
 ## The Executive Director
 The role of the executive director is to manage the day to day
 operations of the SunPy organization. The executive director (ED for
-short) shall report to the board. The executive director shall have
-push privileges to the SunPy repository and can elect SunPy
-maintainers from the developer community to have the same privileges.
-The ED shall have ownership privileges to the sunpy github
-organization. The ED shall hold regular meetings with the developer
-community (developer meeting).
+short) shall:
+
+* hold regular meetings with the developer community (at least monthly)
+* make regular reports to the board (at least quarterly)
+* have push privileges to the SunPy repository
+and can delegate those privileges
```

Comment:
Because this is Markdown, it's already one line (as intended), but I'll go ahead and remove the (non-rendered) newline for super-clarity.